docs/source/quickstart.rst

```diff
@@ -3,117 +3,162 @@
 ****************
 Quickstart Guide
 ****************
+The following sections describe, in short with examples, how to setup a few minimal cocotb testcases, how to run the simulation and how to view the generated waveform. For a more thorough and complete explanations about some of the intricacies of cocotb testbenches refer to the :ref:`writing_tbs`.
 
-In the following sections,
-we are walking you through creating and running a small but complete cocotb testbench
-for a fictional *Design Under Test* (:term:`DUT`) called ``my_design``.
+Prerequisites
+=============
+Before starting, install the :ref:`prerequisites<install-prerequisites>` and
+cocotb itself: *pip install cocotb*
+
+Run *cocotb-config --version* in a terminal window to verify that cocotb is installed.
+
+The examples described are made to work with the `icarus <https://steveicarus.github.io/iverilog/>`_. However, another verilog simulator can probably be used as well. See :ref:`simulator-support` for a comprehensive list of the cocotb supported simulators.
```

Comment:
```suggestion
```
The quickstart code needs to work for all supported simulators (and our CI will make sure that is true).